### PR TITLE
add support for G.4X and G.8X in glue_job.py

### PR DESCRIPTION
**Pull request by randomnaja**
##### SUMMARY
according to https://aws.amazon.com/about-aws/whats-new/2023/05/aws-glue-large-instance-types-generally-available/ I added support for these 2 new instance types

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
glue_job

### CODE REVIEW

File: plugins/modules/glue_job.py [hunk 1]
```diff
@@ -95,7 +95,8 @@
   worker_type:
     description:
       - The type of predefined worker that is allocated when a job runs.
-    choices: [ 'Standard', 'G.1X', 'G.2X' ]
+      - new instance types G.4X and G.8X [https://aws.amazon.com/about-aws/whats-new/2023/05/aws-glue-large-instance-types-generally-available/]
```
Comment (tremble) [inline plugins/modules/glue_job.py:98]:
```suggestion
      - Support for instance types C(G.4X( and C(G.8X) was added in community.aws release 7.2.0.
```

**Review by tremble — CHANGES_REQUESTED** [review]
Thanks for taking the time to submit this PR.

Please add a changelog fragment: https://docs.ansible.com/ansible/latest/community/development_process.html#making-your-pr-merge-worthy

**Comment by randomnaja** [comment]
> Thanks for taking the time to submit this PR.
> 
> Please add a changelog fragment: https://docs.ansible.com/ansible/latest/community/development_process.html#making-your-pr-merge-worthy

I added changelog fragment

**Review by alinabuzachis — APPROVED** [review]

File: changelogs/fragments/2048-add-new-instance-types-in-gluejob.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - glue_job - add support for 2 new instance types which are G.4X and G.8X
```
Comment (tremble) [inline changelogs/fragments/2048-add-new-instance-types-in-gluejob.yaml:2]:
```suggestion
  - glue_job - add support for 2 new instance types which are G.4X and G.8X (https://github.com/ansible-collections/community.aws/pull/2048).
```

**Review by tremble — APPROVED** [review]

**Comment by randomnaja** [comment]
Is there any progress on merging this PR ?

**Pull request merged**